addons/mail/static/src/models/message/message.js

```diff
@@ -612,6 +629,13 @@ function factory(dependencies) {
                 'originThreadIsModeratedByCurrentPartner',
             ],
         }),
+        isSubjectEqualThreadName: attr({
+            compute: '_computeIsSubjectEqualThreadName',
+            dependencies: [
+                'subject',
+                'originThread',
+            ],
+        }),
```

Comment:
you also need a dependency on `originThreadName` (you'll need to make a related field for it)